.github/workflows/ci.yml

```diff
@@ -70,6 +70,8 @@ jobs:
         with:
           ruby-version: '2.7'
           bundler-cache: true
+      - name: Check that all rules are declared in the all rules acceptance test
```

Comment:
Was bit quick to approve here , given this will be overwritten I assume adding a whole new file in this directory will be okay. Maybe that's the best thing to do.